packages/salesforcedx-vscode-lwc/src/messages/i18n.ts

```diff
@@ -23,6 +23,8 @@ export const messages = {
     'To run this command, first install the @salesforce/lwc-dev-server plugin. For more info, see [https://developer.salesforce.com/docs/component-library/documentation/lwc/lwc.get_started_local_dev](https://developer.salesforce.com/docs/component-library/documentation/lwc/lwc.get_started_local_dev).',
   force_lightning_lwc_start_addr_in_use:
     "The local development server couldn't start as the address is already in use. To resolve, do one of these\n 1) Stop the local dev server running on any another instance.\n or 2) Change the default port [Configuration for Projects (Optional)](https://developer.salesforce.com/docs/component-library/documentation/en/lwc/lwc.get_started_local_dev_setup).\n 3) Kill the process running on the specified port.",
+  force_lightning_lwc_inactive_scratch_org:
+    'The local development server couldn’t start as your scratch org is not active. Run "SFDX: Create a Default Scratch Org" to create a scratch org, or run "SFDX: Set a Default Org" to switch to an active scratch org.',
   force_lightning_lwc_start_failed:
     'The local development server was not able to start.',
```

Comment:
The local development server can't start.'

<ekapner>any other help we can provide?

Reply:
This is the default message when we can't determine a cause for server start failure

Reply:
Not sure if we should do more around here. But the Runtime Resiliency epic (which this story is a part of) should mitigate this and locate the reason more precisely.

Reply:
To expand, I don't think this will ever be presented to the user. Like Xiaoyi mentioned, this was added in the event that we couldn't determine the cause of a server start failure. This was added to ensure that the call to start the server would be halted from the extension side in the event that something unexpected occurred.